fixtures/secrets/main.tf

```diff
@@ -0,0 +1,57 @@
+resource "aws_secretsmanager_secret" "tfe_license" {
+  count       = var.tfe_license == null ? 0 : 1
+  description = "The TFE license."
```

Comment:
The `name` attribute of each variable object should be passed to the `name` argument of its respective `aws_secretsmanager_secret`.

Reply:
@sandrampeter to clarify, what I meant was that these values should be set like `name = var.tfe_license.name`.